openstack_controller/datadog_checks/openstack_controller/components/image.py

```diff
@@ -43,9 +43,10 @@ def _report_images(self, config, tags):
                     tags=GLANCE_IMAGE_TAGS,
                     prefix=GLANCE_IMAGE_PREFIX,
                     metrics=GLANCE_IMAGE_METRICS,
-                    lambda_name=lambda key: 'up' if key == 'status' else 'bytes' if key == 'name' else key,
+                    lambda_name=lambda key: 'up' if key == 'status' else 'size' if key == 'name' else key,
```

Comment:
`name` json field is our `size` metric?

Reply:
We shouldn't need to use `lambda_name` and `lambda_value` for the .size metric; I've removed that logic.